modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -862,6 +864,19 @@ public void postProcessModelProperty(CodegenModel model, CodegenProperty propert
         return objs;
     }
 
+    @Override
+    public void addImportsToOneOfInterface(List<Map<String, String>> imports) {
```

Comment:
is this method the same as the one in JavaClientCodegen?

Reply:
@agilob, implementation is the same for the present moment. I am not yet sure that I can extract the implementation to the `DefaultCodegen`.